charts/policies/pod-privileged/questions.yaml

```diff
@@ -0,0 +1,50 @@
+# This is a Rancher questions file
+---
+
+# General policy questions:
```

Comment:
This is the simplest policy example, with a policy that has the configuration general to all policies, and it has no specific policy settings.

It uses some `enums`, all fine.
[simplest-policy-example.webm](https://user-images.githubusercontent.com/2196685/188899364-d7552c5b-2e2c-40ba-b040-419771360254.webm)